_posts/2023-11-20-securing-application-with-other-oidc.adoc

```diff
@@ -0,0 +1,274 @@
+
+Now we can clone the forked elytron-examples repository to your local machine. We will also need to create a new branch, where we will make some changes and commit it to:
+```
+git clone git@github.com:<your_github_username>/elytron-examples.git
+git checkout -b simple-webapp-okta
+cd simple-webapp-okta
+```
+
+== Log Into the OpenShift Cluster
+Before we can deploy our application, we need to log into an OpenShift cluster. You can log in via the OpenShift CLI:
+```
+oc login -u myUserName
+```
+Alternatively, to log in using the API token, navigate to Developer Sandbox, click on your username at the top right corner, copy your login command and paste it on your terminal. The command will be of this format:
+
+```
+oc login --token=myToken --server=myServerUrl
+```
+Once you have your environment set up with the required tools, we can move on to the next step to build and deploy our application on OpenShift.
+
+== Setting Up the Okta Admin Console 
+In order to secure the application, we will need to access the web console. 
+
+. Visit https://www.okta.com/free-trial/ and under `Workforce Identity Cloud`, sign up or log in to get started. 
+. At the top right corner, click on *Admin* to be redirected to the admin dashboard. 
+. Under the *Add an app to use Single Sign-on* option, click on *Add App* to add the application we want to secure. Click on *Create New App*, choose *OIDC - OpenID Connect* under `Sign-in method` and *Web Application* for `Application type` and click `Next`. 
+. Under *General Settings* add the following configurations: 
+* `App integration name:` *simple-webapp-okta*
+* `Grant type:` Check *Client Credentials* 
+* `Sign-in redirect URIs`: Leave it as is for now. We will edit it later.
+* Under the `Assignments` tab, for `Controlled access`, select *Allow everyone in your organization to access* and disable *Federation Broker Mode*. 
+* Click `Save`
+. Navigate to the Directory drop down on the left panel and click `People`. Click *Add person* and add a person with: 
+* First name: *Alice*
+* Last name: *Smith*
+* Email: *alice@example.org*
+* User Name: *alice@example.org* 
+* Check `I will set password` and set a password that fits the password requirements. 
+* Check off `User must change password on first login` and hit `Save`.
+. Go back to your application using the `Application` tab on the left hand panel and under assignments, click *Assign* and *Assign to People* and select *Alice*. For more information about how to assign a user to an application, please see https://support.okta.com/help/s/article/How-To-Assign-An-User-To-An-Application?language=en_US[here] and to learn more about how to add a user manually https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-add-users.htm[click here].
```

Comment:
s/Application/Applications

Reply:
Click on the *Applications* section in the menu on the left side. Then navigate to *Applications* and click on *simple-webapp-okta*. Then under the *Assignments* tab,....

Reply:
I think the part about how to add a user manually should be added to the step where you add the user.

Reply:
fixed!